packages/ruby/lib/readme/har.rb

```diff
@@ -67,7 +67,7 @@ def response_body
       if @response.content_type == "application/json"
         begin
           parsed_body = JSON.parse(@response.body.first)
-          return filter_with_params(parsed_body)
+          return HarCollection.new(@filter, parsed_body).to_h.to_json
```

Comment:
This wouldn't posted to the Metrics API until I called `to_json` on the result of the HarCollection. Should this happen inside the `to_h` method?

Reply:
No it probably shouldn't convert to json inside `to_h` because I can see wanting to turn that hash into other structures such as URL-encoded string or XML

Reply:
We disovered that we we do indeed want to double-encode JSON